lib/at_monitor/at_monitor.c

```diff
@@ -107,6 +120,16 @@ static int at_monitor_sys_init(void)
 {
 	int err;
 
+#if CONFIG_AT_MONITOR_USE_DEDICATED_WORKQUEUE
+	const struct k_work_queue_config cfg = {.name = "at_monitor_work_q"};
+
+	k_work_queue_start(&at_monitor_work_q,
+			   at_monitor_work_q_stack,
+			   K_THREAD_STACK_SIZEOF(at_monitor_work_q_stack),
+			   K_LOWEST_APPLICATION_THREAD_PRIO,
```

Comment:
`CONFIG_AT_MONITOR_WORKQ_PRIO` not taken into account

Reply:
Thanks, updated